src/js/components/Menu/Menu.js

```diff
@@ -444,7 +445,12 @@ const Menu = forwardRef((props, ref) => {
               align.top !== 'bottom'
                 ? controlMirror
                 : undefined}
-              <Box overflow="auto" role="menu" a11yTitle={a11y}>
+              <Box
+                overflow="auto"
+                role="menu"
+                a11yTitle={a11y}
+                {...(!grouped ? theme.menu.container : {})}
```

Comment:
Would it make sense to do something like this

If I have `theme.menu.container` defined and `theme.menu.group.container` is undefined then use use `theme.menu.container` regardless of whether the Menu is grouped or not.

If I have `theme.menu.container` defined and `theme.menu.group.container` is also defined then don't use `theme.menu.container` for grouped Menus.

Reply:
Good comment -- I think that makes sense. Will adjust the logic

Reply:
Done.